pkg/stacks/unpack_test.go

```diff
@@ -531,15 +555,23 @@ metadata:
     stacks.crossplane.io/icon-data-uri: data:image/svg+xml;base64,bW9jay1pY29uLWRhdGEtc3Zn
     stacks.crossplane.io/stack-title: Sample Crossplane Stack
     stacks.crossplane.io/ui-schema: |-
-      version: 0.3
       configSections:
       - title: group Title
         description: group Description
+      resourceConditionedStatusTypes:
+      - Ready
+      stackConditionedStatusTypes:
+      - Ready
+      version: 0.4
       ---
-      version: 0.3
       configSections:
       - title: sibling Title
         description: sibling Description
+      resourceConditionedStatusTypes:
+      - Ready
+      stackConditionedStatusTypes:
+      - Ready
+      version: 0.4
```

Comment:
The diffs would be cleaner if the `version: 0.4` lines replaced the `version: 0.3` lines.  Any reason why the version is being tacked on to the bottom of the examples instead of the top?

Reply:
Really the only reason was just to keep it all alphabetical, I can move them back up if needed though

Reply:
just updated